lib/valueflow.cpp

```diff
@@ -4924,15 +4995,14 @@ struct MultiValueFlowAnalyzer : ValueFlowAnalyzer {
     }
 
     virtual bool isAlias(const Token* tok, bool& inconclusive) const OVERRIDE {
-        std::list<ValueFlow::Value> vals;
-        std::transform(values.begin(), values.end(), std::back_inserter(vals), SelectMapValues{});
+        const auto range = MultiRange(values);
```

Comment:
It would be nicer if we had a lazy transform function we could do instead: `const auto range =  Transform(values, SelectMapValues{})`. There's many place we are calling `std::transform`(which a lot of times it makes a copy), so we could get rid of these copies.

Reply:
Yeah - that would be the goal. But I was happy to just piece this together what actually works...